pkg/kubectl/cmd/util/helpers.go

```diff
@@ -45,7 +46,8 @@ import (
 )
 
 const (
-	ApplyAnnotationsFlag = "save-config"
+	ApplyAnnotationsFlag      = "save-config"
+	ChangeSourceAnnotationKey = "kubectl.kubernetes.io/change-source"
```

Comment:
Can we get a better name than `change-source` for this? Maybe `recorded-command`?